service/src/main/resources/schema/development/schema.json

```diff
@@ -138,6 +138,17 @@
         }
       }
     },
+    "generatedBy": {
+      "type": "object",
+      "properties": {
+        "TerraCore:hasAssayCategory": {
```

Comment:
For these categories, do we have recommended dictionaries or ontologies? I think we want different catalog entries to agree on these terms, so either restricting the list, or providing a list of recommendations would be helpful.

Reply:
@kreinold says, we can remove this requirement / field for now, as we don't think it will be generally useful for searchability.